### PR TITLE
Remove rubygem-less and rubygem-less-rails

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by domcleal** [comment]
Thanks, could you also add these packages to the obsoletes list in tfm/tfm.spec please?  (Seems an OK way to ensure removal of old things from within the tfm collection to me.)

**Comment by ehelms** [comment]
Updated with obsoletes

File: tfm/tfm.spec [hunk 1]
```diff
@@ -62,6 +62,8 @@ Obsoletes: ruby193-ruby-wrapper
 # and to ensure smooth upgrades when old packages deps aren't satisified
 Obsoletes: %{scl_prefix}rubygem-sass < 3.4.9-3
 Obsoletes: %{scl_prefix}rubygem-sprockets < 2.12.3-3
+Obsoletes: %{scl_prefix}less < 2.5.1-4
```
Comment (domcleal) [inline tfm/tfm.spec]: -5, as the latest package in nightly is -4

File: tfm/tfm.spec [hunk 2]
```diff
@@ -62,6 +62,8 @@ Obsoletes: ruby193-ruby-wrapper
...
+Obsoletes: %{scl_prefix}less-rails < 2.5.0-3
```
Comment (domcleal) [inline tfm/tfm.spec]: -4

**Comment by ehelms** [comment]
Corrected

**Comment by domcleal** [comment]
Thanks @ehelms, merged.

**Comment by domcleal** [comment]
Old packages untagged and tfm 2.0-2 built.

**Pull request merged**